recipes/r-srs/meta.yaml

```
@@ -0,0 +1,94 @@
+{% set version = '0.2.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-srs
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/SRS_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/SRS/SRS_{{ version }}.tar.gz
+  sha256: 142382c101f390b40dea0ac75483a844a9667f102c61ea2aa397cf1e92d4a15b
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-dt >=0.16
+    - r-shiny >=1.5.0
+    - r-shinybusy >=0.2.2
+    - r-shinycssloaders >=1.0.0
+    - r-vegan >=2.5_6
+  run:
+    - r-base
+    - r-dt >=0.16
+    - r-shiny >=1.5.0
+    - r-shinybusy >=0.2.2
+    - r-shinycssloaders >=1.0.0
+    - r-vegan >=2.5_6
+
+test:
+  commands:
+    - $R -e "library('SRS')"           # [not win]
+    - "\"%R%\" -e \"library('SRS')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=SRS
+  license: CC BY-SA 4.0
```

According to https://spdx.org/licenses/CC-BY-SA-4.0.html
```suggestion
  license: CC-BY-SA-4.0
```